### ISSUE TITLE
can't update massage binding 

### ISSUE THREAD

**Issue by ligadata-saleh** [LigaData/Kamanja#1089]
Hi , 
   can't update massage binding , i have to remove then add the new massage binding . 

Thanks
Saleh

**Comment by ligadata-pokuri** [comment]
Yes we don't support updating adapter message binding. We are asking to remove & add new one. Need to see whether update make sense or not. In future we may support more than one serializer for same adapter & message.
